docs/source/configuration/persisted-queries.mdx

```diff
@@ -109,6 +109,27 @@ To enable safelisting, you _must_ turn off [automatic persisted queries](./in-me
 
 </Note>
 
+
+#### `local_safelist`
+
+Adding `local_safelist` to your `safelist` configuration allows you to use a local persisted query manifest in place of the hosted Uplink version. This is useful when you have an offline Enterprise license and are unable to use Uplink.
+
+```yaml title="router.yaml"
+persisted_queries:
+  enabled: true
+  safelist:
+    enabled: true
+    local_safelist: ./persisted-query-manifest.json
+apq:
+  enabled: false
+```
+
+<Note>
+
+To enable safelisting, you _must_ turn off [automatic persisted queries](./in-memory-caching#caching-automatic-persisted-queries-apq) (APQs). APQs let clients [register arbitrary operations at runtime](/graphos/operations/persisted-queries/#differences-from-automatic-persisted-queries) while safelisting restricts operations to those that have been explicitly registered.
```

Comment:
I'm not sure why this is included in this section.

Reply:
All of the other safelist sections had it- was just following the same format. But given the first comment- it'll be removed in any case.